pkgdb2/ui/acls.py

```diff
@@ -39,14 +39,16 @@
 # pylint: disable=E1101
 
 
-@UI.route('/acl/<package>/request/', methods=('GET', 'POST'))
+@UI.route('/acl/<namespace>/<package>/request/', methods=('GET', 'POST'))
 @fas_login_required
-def request_acl(package):
+def request_acl(namespace, package):
```

Comment:
Can we do a default ``namespace=rpms`` here like in that one other flask endpoint?